react/features/status-bar/components/index.js

```diff
@@ -0,0 +1,3 @@
+export { default as AudioMutedIcon } from './AudioMutedIcon';
+export { default as ModeratorIcon } from './ModeratorIcon';
+export { default as VideoMutedIcon } from './VideoMutedIcon';
```

Comment:
I had moved the icons to features/filmstrip when resolving the last merge conflicts and didn't properly remove these icons. The ones in features/filmstrip are the correct ones.